Javascript(ES5)/README.md

```diff
@@ -1193,7 +1734,9 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
     this.firstName = 'Panda';
     ```
 
-  - Don't save references to this. Use Function#bind.
+  - Don't save references to this. Use [`Function#bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind).
```

Comment:
Since there was no consensus, I'll leave it like this. (We can always come back and change it later.)